Can't build and test on fresh clone of the repository using latest tsc

## Description

I was trying to build Astra-Monitor on my Fedora workstation. After installing all dependencies  and running `bash ./test.sh` it throws typescript errors related to GLib20 & GLib20. 

## Steps to Reproduce

1. Do the fresh clone
2. Adjust display and resolution according to your setup (`XEPHYR_DISPLAY=":0"`, `RESOLUTION=1280x720`)
3. install `libgtop` and `typescript`
4. npm install 
5. Build using `bash ./test.sh`
## Screenshots

![Image](https://github.com/user-attachments/assets/3aeb0c69-8998-4704-8e92-30bfb4ce31f7)

## Environment

<!-- Please complete the following information -->

-   **Astra Monitor Version**: latest changes on main branch (commit id: `e966412b18bdacb9267c4f0769dd421502caa314`)
-   **GNOME Version**: 47
-   **Operating System and Version**: Fedora Linux 41 (Workstation Edition)
-   **Other Relevant System Information**: Running on Wayland, TypeScript Version 5.7.3

## Logs
```bash
node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20 - error TS2300: Duplicate identifier 'GLib20'.

3     export default GLib20;
                     ~~~~~~

  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20
    3     export default GLib20;
                         ~~~~~~
    'GLib20' was also declared here.
  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20
    8     export default GLib20;
                         ~~~~~~
    and here.

node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20 - error TS2300: Duplicate identifier 'GLib20'.

8     export default GLib20;
                     ~~~~~~

  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20
    3     export default GLib20;
                         ~~~~~~
    'GLib20' was also declared here.
  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20
    8     export default GLib20;
                         ~~~~~~
    and here.

node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20 - error TS2300: Duplicate identifier 'GObject20'.

3     export default GObject20;
                     ~~~~~~~~~

  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20
    3     export default GObject20;
                         ~~~~~~~~~
    'GObject20' was also declared here.
  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20
    8     export default GObject20;
                         ~~~~~~~~~
    and here.

node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20 - error TS2300: Duplicate identifier 'GObject20'.

8     export default GObject20;
                     ~~~~~~~~~

  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20
    3     export default GObject20;
                         ~~~~~~~~~
    'GObject20' was also declared here.
  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20
    8     export default GObject20;
                         ~~~~~~~~~
    and here.

node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20 - error TS2300: Duplicate identifier 'GLib20'.

3     export default GLib20;
                     ~~~~~~

  node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20
    3     export default GLib20;
                         ~~~~~~
    'GLib20' was also declared here.
  node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20
    8     export default GLib20;
                         ~~~~~~
    and here.

node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20 - error TS2300: Duplicate identifier 'GLib20'.

8     export default GLib20;
                     ~~~~~~

  node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3:20
    3     export default GLib20;
                         ~~~~~~
    'GLib20' was also declared here.
  node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:8:20
    8     export default GLib20;
                         ~~~~~~
    and here.

node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-import.d.ts:5:9 - error TS2717: Subsequent property declarations must have the same type.  Property 'GLib' must be of type 'typeof GLib', but here has type 'typeof GLib'.

5         GLib: typeof GLib20;
          ~~~~

  node_modules/@girs/glib-2.0/glib-2.0-import.d.ts:5:9
    5         GLib: typeof GLib20;
              ~~~~
    'GLib' was also declared here.

node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20 - error TS2300: Duplicate identifier 'GObject20'.

3     export default GObject20;
                     ~~~~~~~~~

  node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20
    3     export default GObject20;
                         ~~~~~~~~~
    'GObject20' was also declared here.
  node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20
    8     export default GObject20;
                         ~~~~~~~~~
    and here.

node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20 - error TS2300: Duplicate identifier 'GObject20'.

8     export default GObject20;
                     ~~~~~~~~~

  node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3:20
    3     export default GObject20;
                         ~~~~~~~~~
    'GObject20' was also declared here.
  node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:8:20
    8     export default GObject20;
                         ~~~~~~~~~
    and here.

node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-import.d.ts:5:9 - error TS2717: Subsequent property declarations must have the same type.  Property 'GObject' must be of type 'typeof GObject', but here has type 'typeof GObject'.

5         GObject: typeof GObject20;
          ~~~~~~~

  node_modules/@girs/gobject-2.0/gobject-2.0-import.d.ts:5:9
    5         GObject: typeof GObject20;
              ~~~~~~~
    'GObject' was also declared here.


Found 10 errors in 6 files.

Errors  Files
     2  node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3
     2  node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3
     2  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-ambient.d.ts:3
     1  node_modules/@girs/gtop-2.0/node_modules/@girs/glib-2.0/glib-2.0-import.d.ts:5
     2  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-ambient.d.ts:3
     1  node_modules/@girs/gtop-2.0/node_modules/@girs/gobject-2.0/gobject-2.0-import.d.ts:5
[2025-02-03 21:11:15] Failed to compile TypeScript
```

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
         "eslint": "^8.57.0",
         "@girs/gnome-shell": "^47.0.0-next.4",
         "@girs/gjs": "^4.0.0-beta.16",
-        "@girs/gtop-2.0": "2.0.0-4.0.0-beta.16",
+        "@girs/gtop-2.0": "^2.0.0-4.0.0-beta.19",
         "@typescript-eslint/eslint-plugin": "^7.7.1",
         "@typescript-eslint/parser": "^7.7.1",
         "prettier": "^3.2.5"

